hazelcast/src/main/java/com/hazelcast/cache/impl/DeferredValue.java

```diff
@@ -71,8 +71,12 @@ public Data getSerializedValue(SerializationService serializationService) {
         return serializedValue;
     }
 
-    // returns a new DeferredValue representing the same value as this
     public DeferredValue<V> shallowCopy() {
+        return shallowCopy(true, null);
+    }
+
+    // returns a new DeferredValue representing the same value as this
+    public DeferredValue<V> shallowCopy(boolean resolved, SerializationService serializationService) {
```

Comment:
Can you add javadoc explaining what `resolved` is about?

Reply:
done